telos-frontend/src/components/calendar/DashboardCalendar.jsx

```diff
@@ -1,3 +1,35 @@
-const DashboardCalendar = () => <div />;
+import { Calendar, momentLocalizer } from 'react-big-calendar';
+import moment from 'moment';
+import './react-big-calendar.css';
```

Comment:
Try using `import 'react-big-calendar/lib/css/react-big-calendar.css'` instead of `import './react-big-calendar.css'`
You shouldn't need to copy the components css file onto here

Reply:
Yeah, thats true, all done. Thanks!